test/testRunner.c

```diff
@@ -0,0 +1,11 @@
+#include "ventilation-test.h"
+#include "testUtil.h"
+
+int main(int argc, char **argv) {
+  //test(addition, "addition");
+  test(startVentilation_openInspiratoryValve, "startVentilation_openInspiratoryValve");
+  test(startVentilation_doesNotopenInspiratoryValve, "startVentilation_doesNotopenInspiratoryValve");
+    
+}
+
+
```

Comment:
This file is named using camel case (`testRunner`) but other files are named using snake case (`test_runner`). Please pick one and stick to it.

Personally, I think all files should be named using snake case (`test_runner.c`).